api/container/containerv2/alb.go

```diff
@@ -88,10 +193,108 @@ func (r *alb) GetAlb(albID string, target ClusterTargetHeader) (AlbConfig, error
 	return successV, err
 }
 
+func (r *alb) UpdateAlb(updateAlbReq UpdateALBReq, target ClusterTargetHeader) error {
```

Comment:
This func may miss a comment/doc. Also in the example there is a call for ChangeALBUpdatePolicy to disable auto update. Is it mandatory? should this be done here as well?

Reply:
Fixed